networking/networking.go

```diff
@@ -229,23 +233,43 @@ func Load(podRoot string, podID *types.UUID) (*Networking, error) {
 	}, nil
 }
 
-func (n *Networking) GetDefaultIP() net.IP {
-	if len(n.nets) == 0 {
-		return nil
+// GetDefaultNet iterates through all loaded networks and returns either
+// the first network that has masquerading enabled, the last network in case
```

Comment:
Oh, this is totally by mistake and I didn't commit the correct lines. Thanks for catching.